### PR TITLE
Grafana config 

### PR DESCRIPTION
**Pull request by suiguoxin**

### CODE REVIEW

**Review by Binyang2014 — COMMENTED** [review]
Can we combine the normal task/node with with pxc task/node view? Or you can create a new dashboard to show this

File: src/grafana/deploy/grafana-configuration/pai-jobview-dashboard.json.template [hunk 1]
```diff
@@ -227,14 +227,14 @@
               "label": "GB",
               "logBase": 1,
               "max": null,
-              "min": null,
+              "min": 0,
```
Comment (Binyang2014) [inline src/grafana/deploy/grafana-configuration/pai-jobview-dashboard.json.template:230]: What's the difference for 0 and null?

Comment (suiguoxin) [reply src/grafana/deploy/grafana-configuration/pai-jobview-dashboard.json.template:230]: The yaxis may begin from negative values if set as null.

**Comment by suiguoxin** [comment]
> Can we combine the normal task/node with with pxc task/node view? Or you can create a new dashboard to show this

Combined in the same dashboard, the queries without data will not be shown.

**Review by Binyang2014 — APPROVED** [review]

**Pull request merged**